Core/Core/Api/GraphQL/Enums/FileUploadConversionStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Speckle.Core.Api.GraphQL.Enums;
+
+//TODO: This enum isn't explicitly defined in the schema, instead its usages are int typed (But represent an enum)
+public enum FileUploadConversionStatus
+{
+  Queued,
```

Comment:
Not sure I understand what you mean...
I will remove the `TODO` part of the comment (that was more just a friendly reminder for me to double check that the comment was indeed accurate and true)

Reply:
If we rely on Queued being 0 and never changing, i.e. because it is serialised, then we should be explicit about it and add a comment to say, do NOT change this enum, IF additions are required then please add after